Resources/Prototypes/Traits/physical.yml

```diff
@@ -521,7 +507,7 @@
           action: ActionSericulture
           productionLength: 2
           entityProduced: MaterialWebSilk1
-          hungerCost: 4
+          hungerCost: 1 
```

Comment:
If you really want to buff this, consider making this trait also add the tag needed by web crafting recipes? Actually I kinda forgot if we ever put in a TraitAddTag, but it would be kinda trivial to add I think.